server/swagger/signin.swagger.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @swagger
+ * /api/v1/users/signin:
+ *   post:
+ *     tags:
+ *       - auth
```

Comment:
I think it's ok.
However, if you could change this to "Authentication" so that we have all auth concerned in the same section

Reply:
It's done.